Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3233,9 +3235,9 @@ class BlurWrapper: UIView {
             }
             effectView.hidden = disableBlur
             switch blurStyle {
-            case .ExtraLight, .Light:
+            case .ExtraLight, .Light, .Regular, .Prominent:
```

Comment:
I'm not sure what these `Regular` and `Prominent` styles are, and I can't find much documentation about them. I arbitrarily bucketed them into the light style, but let me know if you can figure out what these are...